src/test/java/posters/pageobjects/components/MiniCart.java

```diff
@@ -38,22 +38,22 @@ public class MiniCart extends AbstractComponent
     @Step("ensure availability mini cart")
     public void isComponentAvailable()
     {
-        $("#btnCartOverviewForm").should(exist);
+        $("#header-customer-menus").should(exist);
     }
 
     /// ----- mini cart navigation ------ ///
 
     @Step("open the mini cart")
     public void openMiniCart()
     {
-        headerCart.hover();
```

Comment:
the mini cart still opens on hover on the page is there a reason, this was removed?
EDIT: I noticed that click also opens the mini cart and pins it. So we could do both ways.